packages/libzip/project.bri

```diff
@@ -0,0 +1,72 @@
+import * as std from "std";
+import { cmakeBuild } from "cmake";
+import nushell from "nushell";
+import openssl from "openssl";
+
+export const project = {
+  name: "libzip",
+  version: "1.11.4",
+};
+
+const source = Brioche.download(
+  `https://libzip.org/download/libzip-${project.version}.tar.gz`,
+)
+  .unarchive("tar", "gzip")
+  .peel();
+
+export default function libzip(): std.Recipe<std.Directory> {
+  return cmakeBuild({
+    source,
+    dependencies: [std.toolchain, openssl],
+    set: {
+      BUILD_REGRESS: "OFF",
+      BUILD_EXAMPLES: "OFF",
+    },
+  }).pipe((recipe) =>
+    std.setEnv(recipe, {
+      CPATH: { append: [{ path: "include" }] },
+      LIBRARY_PATH: { append: [{ path: "lib" }] },
+      PKG_CONFIG_PATH: { append: [{ path: "lib/pkgconfig" }] },
+      CMAKE_PREFIX_PATH: { append: [{ path: "lib/cmake" }] },
```

Comment:
#nonblocking The question here is also relevant: https://github.com/brioche-dev/brioche-packages/pull/604#pullrequestreview-2894655686

...but I'm fine getting this in and figuring that out as a follow-up